common-libs/tools-openapi/src/helpers/client.ts

```diff
@@ -62,7 +62,7 @@ type GetRequestOptions<TOperation extends MethodTypeByOperation<any>> = Paramete
 export type GetParams<TOperation extends MethodTypeByOperation<any>> = GetRequestOptions<TOperation>['params']
 
 const parseSpec = (rawSpec: GenericApiSpec) => {
-  const basePath = rawSpec.servers[0].url
+  const basePath = rawSpec.servers[0].url === '/' ? '' : rawSpec.servers[0].url
```

Comment:
Maybe we should always remove final `/` just in case?
```
rawSpec.servers[0].url.endsWith('/')
    ? rawSpec.servers[0].url.slice(0, -1)
    : rawSpec.servers[0].url
```

Reply:
Not sure it is a good idea. What if it will be `/api/v1/`, and the methods themselves will have urls like `users/{id}`? In this case we would want to preserve the slash.
Domain root is a special case.

Reply:
I see, I thought we're consistent with this, but maybe it's not the case.